### PR TITLE
Add minimum and default for the `Condenser Minimum Flow Fraction` field

### PR DESCRIPTION
**Pull request by lymereJ**
Pull request overview
---------------------
 - Address [this ](https://github.com/NREL/EnergyPlus/pull/10511#pullrequestreview-2342091814)review comment.

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

File: idd/Energy+.idd.in [hunk 1]
```diff
@@ -73676,6 +73676,8 @@ Chiller:Electric:EIR,
        \note corresponds to this fraction multiplied by the maximum condenser flow rate. This input is only used
        \note when the "Condenser Flow Control" input is set to "ModulatedChillerPLR", "ModulatedLoopPLR" or
        \note "ModulatedDeltaTemperature".
+       \minimum> 0.0
+       \default 0.2
```
Comment (jmarrec) [inline idd/Energy+.idd.in]: Thanks @lymereJ 

Just checking: is 0.0 not a valid value?

**Review by jmarrec — APPROVED** [review]

File: idd/Energy+.idd.in (hunk 1 above)
Comment (jmarrec) [reply idd/Energy+.idd.in]: And does > 1 make sense here? Otherwise `\maximum 1` would be good?

Comment (lymereJ) [reply idd/Energy+.idd.in]: The minimum was copy/paste typo, good catch. I'll add the maximum.

**Comment by nrel-bot-2** [comment]
@lymereJ @myoldmopar it has been 28 days since this pull request was last updated.

**Comment by Myoldmopar** [comment]
This looks quite reasonable.  Thanks @lymereJ, merging this shortly.

**Pull request merged**